SayTheirNames/Resources/Base.lproj/Localizable.strings

```diff
@@ -41,3 +41,4 @@
 "slack_url" = "https://saytheirnames.slack.com/join/shared_invite/zt-eqjuatz7-fgh3zPRXIKiiXsC1Vf3oZA#/";
 "more_history.title" = "SOME HISTORY ON THE SAY THEIR NAME MOVEMENT";
 "massive_thank_you" = "A MASSIVE THANK YOU TO EVERYONE INVOLVED";
+"viewGoFundMePage" = "VIEW GOFUNDME PAGE";
```

Comment:
I think other strings in this file are following a snake_case approach instead of a CamelCase. Maybe we should change it to make it more consistent?

Reply:
@Angel5215 : will do. Thanks !